jwala-services/src/main/java/com/cerner/jwala/service/media/impl/MediaServiceImpl.java

```diff
@@ -78,10 +83,12 @@ public JpaMedia create(final Map<String, String> mediaDataMap, final Map<String,
         final String filename = Paths.get((String) mediaFileDataMap.get("filename")).getFileName().toString();
 
         try {
-            mediaDao.find(media.getName());
-            final String msg = MessageFormat.format("Media already exists with name {0}", media.getName());
-            LOGGER.error(msg);
-            throw new MediaServiceException(msg);
+            JpaMedia media1 = mediaDao.find(media.getName());
```

Comment:
Do not name variables by adding numbers on the end. Please rename media1 to something more meaningful,  such as existingMedia.